docker-compose.yml

```diff
@@ -367,6 +369,37 @@ services:
               --min_confs 0 --local_amt=1000000000 --push_amt=500000000
           fi
         "
+  litd:
+    container_name: litd
+    image: lightninglabs/lightning-terminal:v0.12.4-alpha
+    profiles:
+      - payments
+    restart: unless-stopped
+    healthcheck:
+      <<: *healthcheck
+      test: ["CMD", "curl", "-f", "http://localhost:8443"]
+    depends_on:
+      stacker_lnd:
+        condition: service_healthy
+        restart: true
+    volumes:
+      - stacker_lnd:/lnd
+    ports:
+      - "8443:8443"
+    command:
+      - 'litd'
+      - '--httpslisten=0.0.0.0:8444'
+      - '--insecure-httplisten=0.0.0.0:8443'
+      - '--uipassword=password'
+      - '--lnd-mode=remote'
+      - '--network=regtest'
+      - '--remote.lit-debuglevel=debug'
+      - '--remote.lnd.rpcserver=stacker_lnd:10009'
+      - '--remote.lnd.macaroonpath=/lnd/data/chain/bitcoin/regtest/admin.macaroon'
+      - '--remote.lnd.tlscertpath=/lnd/tls.cert'
+      - '--autopilot.disable'
+      - '--pool.auctionserver=test.pool.lightning.finance:12010'
+      - '--loop.server.host=test.swap.lightning.today:11010'
```

Comment:
Weird, Github must have swallowed my comment here yesterday.

I tried to setup LNC by running the commands provided in the help but couldn't get it to work:

```
f72080b352ec:/# litcli accounts create --balance 100000
[litcli] unable to read macaroon path : open /root/.lit/mainnet/lit.macaroon: no such file or directory
f72080b352ec:/# litcli --macaroonpath /lnd/data/chain/bitcoin/regtest/admin.macaroon accounts create --balance 100000
[litcli] rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: tls: first record does not look like a TLS handshake"
f72080b352ec:/# litcli --macaroonpath /lnd/data/chain/bitcoin/regtest/admin.macaroon --tlscertpath /lnd/tls.cert accounts create --balance 100000
[litcli] rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: tls: first record does not look like a TLS handshake"
```

Am I doing something wrong?

Reply:
`litd` is passed connection args on start. To run `litcli` and have it communicate with `lnd` you need to tell it where it is. Otherwise it assumes it's built-in.

Reply:
Ahh, you mean built-in as in [integrated mode](https://docs.lightning.engineering/lightning-network-tools/lightning-terminal/run-litd#integrated-mode)? I see

Reply:
yep

Reply:
So the issue is these command need `--rpcserver localhost:8444` because the default port doesn't have tls.

I'm going to push an `sndev stacker_litcli` command that does this in a minute.